elliottlib/runtime.py

```diff
@@ -21,7 +21,7 @@
 import assertion
 import exectools
 from pushd import Dir
-from image import ImageMetadata
+from imagecfg import ImageMetadata
```

Comment:
should the `ImageMetadata` be removed from `elliottlib/image.py` then?

Reply:
My intention was rename `image.py` -> `imagecfg.py` to consist with `rpmcfg.py` 
and `ImageMetadata` is still in use here: `meta = ImageMetadata(self, data_obj)`